src/native/corehost/bundle/info.cpp

```diff
@@ -108,7 +108,14 @@ char* info_t::config_t::map(const pal::string_t& path, const location_t* &locati
     // * There is no performance limitation due to a larger sized mapping, since we actually only read the pages with relevant contents.
     // * Files that are too large to be mapped (ex: that exhaust 32-bit virtual address space) are not supported. 
 
+#ifdef _WIN32
+    // Since we can't use in-situ parsing on Windows, as JSON data is encoded in
+    // UTF-8 and the host expects wide strings.
```

Comment:
Is this part of the comment related to the code below on line 115 or above line 88?

Reply:
This is about 115 -- the RapidJson parser used in `json_parser.cpp` modifies the buffer on Linux, but not on Windows, so the CoW permissions are not useful on Windows.